src/subscription/components/SubscribeFormField/index.tsx

```diff
@@ -20,7 +20,7 @@ const SubscribeFormField = () => {
               placeholder={EMAIL_CONTROL.EMAIL_PLACEHOLDER}
               {...field}
               value={field.value || ""}
-              className={`rounded-[10px] text-[16px] focus-visible:ring-transparent ${formState.errors.email ? "border-error" : ""}`}
+              className={`rounded-[10px] text-[16px] focus-visible:ring-transparent h-[48px] ${formState.errors.email ? "border-error" : ""}`}
```

Comment:
```suggestion
 className={cn("rounded-[10px] text-[16px] focus-visible:ring-transparent h-[48px]" 
 , `${formState.errors.email &&"border-error`)}
```
삼항연산자 안쓰고도 가능할것같습니다! 클래스 너무 길어지면 cn으로 감싸는것도 좋을것 같아요!

Reply:
cn 으로 앞으로 최대한 사용하겠습니당 ㅎㅎ
https://github.com/YAPP-Github/24th-Web-Team-1-FE/pull/87/commits/2092e49de50e21536a86bcfe3d1a94fafa1e8bd8